### PR TITLE
Added Kurdish translation files

### PR DESCRIPTION
**Pull request by fadkeabhi**
Added translation in Kurdish language

### CODE REVIEW

File: packages/notification-center/src/i18n/lang.ts [hunk 1]
```diff
@@ -25,6 +25,7 @@ import { CS } from './languages/cs';
 import { PU } from './languages/pu';
 import { TA } from './languages/ta';
 import { KO } from './languages/ko';
+import { KU } from './languages/ki';
```
Comment (p-fernandez) [inline packages/notification-center/src/i18n/lang.ts:28]:
```suggestion
import { KU } from './languages/ku';
```
File name should be `ku.ts`.

File: packages/notification-center/src/i18n/languages/ku.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { ITranslationEntry } from '../lang';
+
+export const KU: ITranslationEntry = {
+  translations: {
+    notifications: 'ئاگادارکردنەوەکان',
```
Comment (p-fernandez) [inline packages/notification-center/src/i18n/languages/ku.ts:5]: This line gets translated to `Warnings` when reversing. Maybe it is better to find a more appropriate translation.

File: packages/notification-center/src/i18n/languages/ku.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+    markAllAsRead: 'هەموویان وەک خوێندراوێک نیشانە بکە',
+    poweredBy: 'بەهێزکراوە لەلایەن',
+    settings: 'دانانەوە',
```
Comment (p-fernandez) [inline packages/notification-center/src/i18n/languages/ku.ts:8]: This line gets translated to `installation` when reversing. Maybe it is better to find a more appropriate translation.

**Review by p-fernandez — CHANGES_REQUESTED** [review]

**Comment by fadkeabhi** [comment]
There are no specific words for notifications and settings in Kurdish (somali) language. So given the words with nearest meaning.

**Review by p-fernandez — APPROVED** [review]

**Pull request merged**